src/index.js

```diff
@@ -14,6 +15,12 @@ import NoticePolicySettings from './settings/NoticePolicy';
 import RequestPolicySettings from './settings/RequestPolicy';
 
 export const settingsRoutes = [
+  {
+    route: 'loan-history',
+    label: 'ui-circulation.settings.index.loanHistory',
+    component: LoanHistory,
+    perm: 'ui-circulation.settings.loan-history',
```

Comment:
Please add this permission to `package.json` so that it is possible to grant it to other users.

Reply:
done